test/autodetect.test.js

```diff
@@ -0,0 +1,87 @@
+/*jslint node: true */
+/*global describe, it, beforeEach, afterEach */
+"use strict";
+
+var chai = require('chai');
+var expect = chai.expect;
+var rewire = require('rewire');
+var cncserver_mod = rewire("../cncserver.js");
+
+var cncserver = cncserver_mod.__get__("cncserver");
+var autoDetectPort = cncserver_mod.__get__("autoDetectPort");
+
+describe('autoDetectPort', function(){
+
+  function setPlatform(mockPlatform) {
+    Object.defineProperty(process, 'platform', {
+      value: mockPlatform
+    });
+  }
+
+  beforeEach( function(){
+    cncserver.gConf.set('serialPath', "{auto}");
+
+    // save original process.platform
+    this.originalPlatform = Object.getOwnPropertyDescriptor(process, 'platform');
+  });
+
+  afterEach( function(){
+    // restore original process.platfork
+    Object.defineProperty(process, 'platform', this.originalPlatform);
+  });
+
+
+  // Test example botConf configuration
+  var botConfController = { manufacturer: "SchmalzHaus",
```

Comment:
Should bring `manufacturer` here down to the next line, two space indent till the end object `}` .